packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.types.ts

```diff
@@ -185,6 +185,11 @@ export interface IComboBoxProps extends ISelectableDroppableTextProps<IComboBox>
    */
   keytipProps?: IKeytipProps;
 
+  /**
+   * Optional mode indicates if arrow keys are valid input when disabled.

```

Comment:
What does "valid" mean here?

Reply:
Accepted. Not ignored, as they are now.

Reply:
It's not really "accepted" it's more handling and more specifically cancelling arrow keys